Problem compiling build

`Library/PackageCache/com.brunomikoski.animationsequencer@0.1.1/Scripts/Runtime/Core/DOTweenActions/DOTweenActionBase.cs(2,10): error CS0234: The type or namespace name 'DOTweenEditor' does not exist in the namespace 'DG' (are you missing an assembly reference?)`

DOTween v.1.2.420 [release build]
Unity v. 2020.1.7f1

--- a/CHANGELOG.MD
+++ b/CHANGELOG.MD
@@ -5,6 +5,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## [0.1.2]
+### Added
+- Added `Rewind()` functionality, thanks @qwe321
+
+### Changed
+- Fixed DOTween editor been used on runtime platform
+
 ## [0.1.1]
 ### Added 
 - Added Complete method for the Sequence, so you can force one sequence to finished instantaneous
@@ -19,10 +26,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - First initial working version
 
 ### [Unreleased]
-   
-  
 
-[0.1.0]: https://github.com/brunomikoski/Animation-Sequencer/releases/tag/v0.1.0
+
+
+[0.1.2]: https://github.com/brunomikoski/Animation-Sequencer/releases/tag/v0.1.2
 [0.1.1]: https://github.com/brunomikoski/Animation-Sequencer/releases/tag/v0.1.1
+[0.1.0]: https://github.com/brunomikoski/Animation-Sequencer/releases/tag/v0.1.0
 
 

--- a/Scripts/Runtime/Core/DOTweenActions/DOTweenActionBase.cs
+++ b/Scripts/Runtime/Core/DOTweenActions/DOTweenActionBase.cs
@@ -1,11 +1,12 @@
 ﻿using System;
+#if UNITY_EDITOR
 using DG.DOTweenEditor;
+#endif
 using DG.Tweening;
 using DG.Tweening.Core;
 using DG.Tweening.Plugins.Core.PathCore;
 using DG.Tweening.Plugins.Options;
 using UnityEngine;
-using Object = UnityEngine.Object;
 
 namespace BrunoMikoski.AnimationSequencer
 {

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "com.brunomikoski.animationsequencer",
   "displayName": "Animation Sequencer",
-  "version": "0.1.1",
+  "version": "0.1.2",
   "unity": "2018.4",
   "description": "Animation sequencer is a way to create complex animations of sequence of events by a friendly user interface",
   "keywords": [

